packages/mdx/test/index.test.js

```diff
@@ -379,7 +387,7 @@ export default function MDXContent({ components, ...props }) {
       name=\\"wrapper\\"
       components={components}>
       <MDXLayout {...layoutProps} {...props}>
-      
+
```

Comment:
This was an issue with VSCode. I disabled "Trim Trailing Whitespace", but it insisted on deleting this line every time I saved the file... The test passes, so hopefully this is fine.

Reply:
Yeah we really need to eventually nuke this flakey test and come up with better assertions rather than a snapshot test.